# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/frontend/BytecodeEmitter.h
# Commit: 1b8c42577e24
# Full Hash: 1b8c42577e2410dbbe050bb30b0ccced9d7c3771
# Author: Ashley Hauck <khyperia@mozilla.com>
# Date: 2019-03-20 10:52:37
# Regressor Bug: 1535166
# File Overlap Count: 1
# Description:
#   Bug 1535166 - Implement computed field names. r=jorendorff
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D23408
# ==============================================================================

diff -r 28370bf14b97 -r 1b8c42577e24 js/src/frontend/BytecodeEmitter.h
--- a/js/src/frontend/BytecodeEmitter.h	Tue Mar 19 22:02:49 2019 +0000
+++ b/js/src/frontend/BytecodeEmitter.h	Tue Mar 19 20:23:03 2019 +0000
@@ -607,6 +607,7 @@
                                      PropListType type);
 
   FieldInitializers setupFieldInitializers(ListNode* classMembers);
+  MOZ_MUST_USE bool emitCreateFieldKeys(ListNode* obj);
   MOZ_MUST_USE bool emitCreateFieldInitializers(ListNode* obj);
 
   // To catch accidental misuse, emitUint16Operand/emit3 assert that they are